docs/user-guide/command-line-interface.md

```diff
@@ -39,6 +39,7 @@ Options:
   --no-color                  Disable color in piped output
   -o, --output-file path::String  Specify file to write report to
   --quiet                     Report errors only - default: false
+  --max-warnings Number       Maximum warning count before eslint exits with an error (or -1 for no limit)
```

Comment:
@ilyavolodin Noticed that too. Some of that is due to emulating optionator's output (but inconsistently). Anything particularly ugly about what I added or could we create a new issue?

@IanVS None whatsoever, unless you had an `eslint` line in a build script and wanted to be explicit about not setting a threshold. To be completely honest, though, it's because I was trying a few different type configs and was unhappy with `Maybe Number` (with null default), `Number | Boolean` (with false default), and a few others I can't remember. I decided to own the `Number` choice and very briefly explain the default. Suggestions welcome re: improving that.